integration-tests/src/test/java/nl/quintor/studybits/ExchangeIT.java

```diff
@@ -0,0 +1,129 @@
+package nl.quintor.studybits;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.ResponseBody;
+import nl.quintor.studybits.enums.ExchangeApplicationState;
+import nl.quintor.studybits.enums.ExchangePositionState;
+import nl.quintor.studybits.models.ExchangeApplicationModel;
+import nl.quintor.studybits.models.ExchangePositionModel;
+import nl.quintor.studybits.models.SchemaDefinitionModel;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class ExchangeIT extends BaseIT {
+    private String UNIVERSITY_NAME = "rug";
+    private String EXCHANGE_UNIVERSITY_NAME = "gent";
+    private String STUDENT_NAME = "lisa";
+    private String ADMIN_NAME = "admin2-gent";
+
+    @Before
+    public void setUp() {
+        super.setUp();
+
+        setupStudentUniversityTriangle(STUDENT_NAME, UNIVERSITY_NAME, EXCHANGE_UNIVERSITY_NAME);
+    }
+
+    @Test
+    public void testCreateExchangePosition() {
+        createExchangePosition();
+        assert getAllExchangePositionModels().size() > 0;
```

Comment:
Please use junit asserts instead of java asserts (this will cause an AssertionError, leading to an errorred test, rather than a failed test.